docs/edge-authorizers/deployment-and-operation.mdx

```diff
@@ -17,18 +17,23 @@ discovery call to the Aserto control plane.
 ## Edge Authorizer Flavors
 
 :::danger Note
-The `onebox` authorizer flavor has been deprecated. Please use the `sidecar`.
+The `sidecar` authorizer is replaced by the [topaz](https://github.com/aserto-dev/topaz).
 :::
 
+
 Currently the following flavors of edge authorizers are available:
 
-- Sidecar: a single-tenant, single policy authorizer without a web UI. The sidecar is the general edge authorizer flavor with a small surface area, which can be used as a sidecar but also as a local service instance.
+- Topaz: a single-tenant, single policy authorizer without a web UI. Topaz is the general edge authorizer flavor with a small surface area, which can be used as a sidecar but also as a local service instance.
 
 ## Docker images
 
 | Flavor  | Image
 | ------- | --------------------------------------------------------
-| Sidecar | docker pull ghcr.io/aserto-dev/sidecar:latest
+| Topaz   | docker pull ghcr.io/aserto-dev/topaz:latest
+
+::: danger Nore
+The topaz images that support being deployed as an edge authorizer > 0.25.5 
```

Comment:
Only Topaz images with a version of 0.25.6 or higher support being deployed as an edge authorizer.